myocamlbuild.ml

```diff
@@ -941,7 +941,7 @@ dispatch
                         "gettext" 
                         (BaseEnvLight.load
                            ~allow_empty:true
-                           ~filename:(Lazy.force MyOCamlbuildBase.env_filename)
+                           ~filename:(MyOCamlbuildBase.env_filename)
```

Comment:
Had to do this to compile oasis. No idea why. If it is wrong, I can revert this change.